bin/videojs-generator-cleanup.js

```diff
@@ -34,51 +43,140 @@ var configs = {
       'scripts.watch:js',
       'scripts.watch:test'
     ]
+  },
+  v3: {
+    removals: [
+      '.babelrc',
+      'dist',
+      'docs/api',
+      'es5',
+      'jsdoc.json',
+      'scripts',
```

Comment:
we may want to specify specific scripts here, just in case they are using scripts of their own in this directory.